ecl/eclcc/eclcc.cpp

```diff
@@ -2106,6 +2081,12 @@ int EclCC::parseCommandLineOptions(int argc, const char* argv[])
         {
             setDebugOption("syntaxCheck", tempBool);
         }
+        else if (iter.matchOption(optMaxErrors, "--maxErrors"))
+        {
+        }
+        else if (iter.matchOption(optUnsuppressImediateSyntaxErrors, "--unsuppressImediateSyntaxErrors"))
```

Comment:
It helped to check syntax errors in the examples to those being counted, I thought it may be useful in the future for other internal tests etc regarding the syntax errors. It was a change I used in the previous work on the error message improvements. I haven't noted it in the usage as it's more for dev use. Let me know if you want it removed.